### PR TITLE
files

### PR DESCRIPTION
**Pull request by mkp246**

### CODE REVIEW

**Comment by mkp246** [comment]
@sweta-singh162  please review

# First ever PR to be reviewed by you

**Comment by sweta-singh162** [comment]
looks interesting 

> can simplify further

**Comment by sweta-singh162** [comment]
look at the text... a woman is like a tea bag :P :8ball:

**Comment by sweta-singh162** [comment]
merged since i wanted to get the feel :dancer:

**Comment by mkp246** [comment]
@sweta-singh162  pit your comments directly in the diff at specific line 
A + sign will be shown when you hover on the line.
**suggestions please**

**Comment by mkp246** [comment]
How u inserted smiley i can only find 5 smileys there.
Thanks you

**Comment by mkp246** [comment]
Change the dp 
🕊 You can see the file with _**someone's name**_  which i didn't show you.

**Comment by sweta-singh162** [comment]
haha... smiley queen hu me to :dancer: :dancers: :sunglasses:

File: wasteland.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+What RE ROOTS THAT CLUTCH, what branches grow
+Out of this rubbish story
+A woman is like a tea bag - you can't tell how strong she is until you put her in hot waterFor every good reason there is to lie, there is a better reason to tell the truth
```
Comment (sweta-singh162) [inline wasteland.txt:3]: :large_blue_diamond: :octocat: :octopus:

**Comment by mkp246** [comment]
heartocat

**Comment by sweta-singh162** [comment]
image hi insert kar diya :laughing:

**Comment by mkp246** [comment]
I will try the one I know and works everywhere 💃 
BTW i got it.
You start typing git will help you out :lol
:Cat

**Comment by sweta-singh162** [comment]
start after a "colon" - ":" like :+1:

**Comment by mkp246** [comment]
🍭   😸 👍

**Pull request merged**